nano/templates/accounting.tmpl

```diff
@@ -0,0 +1,92 @@
+<script type="text/javascript">
+    function markGreen(){
+      var toColor = document.getElementById('diffinput');
+      toColor.style.backgroundColor = "#DDFFDD";
+      var elem = document.getElementById("regid");
+      var body = $('body');
+      elem.value = body.data('urlParameters')["src"];
+	  toColor.focus();
+    }
```

Comment:
All indented with spaces (except the first, but not second, indent level on line 8). Also second indent level is only two spaces.